pkg/volume/azure_file/azure_file.go

```diff
@@ -104,6 +105,13 @@ func (plugin *azureFilePlugin) newMounterInternal(spec *volume.Spec, pod *v1.Pod
 	if err != nil {
 		return nil, err
 	}
+	nameSpace := pod.Namespace
+	if spec.PersistentVolume != nil {
+		ns, err := getSecretNamespace(plugin.host.GetKubeClient(), spec.PersistentVolume)
```

Comment:
I'm referring to the information flow. This has the mounter resolving a storage class from a PV, which isn't done today.